Makefile

```diff
@@ -1,19 +1,18 @@
 SRCPATH		:= $(shell pwd)
-OS_TYPE		:= $(shell ./scripts/ostype.sh)
-   ARCH		:= $(shell ./scripts/archtype.sh)
-PKG_DIR		= $(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH)
+VERSION		:= $(shell $(SRCPATH)/scripts/compute_build_number.sh)
+PKG_DIR		= $(SRCPATH)/packages/$(VERSION)
 
 # This is the default target, build the indexer:
-cmd/algorand-indexer/algorand-indexer:	idb/setup_postgres_sql.go importer/protocols_json.go .PHONY
+cmd/algorand-indexer/algorand-indexer: idb/setup_postgres_sql.go importer/protocols_json.go .PHONY
```

Comment:
Why did this change? It looks like a tab was replaced with a space

Reply:
I had changed that to match the other targets that have spaces there.  Is the tab meaningful there?

Reply:
Dunno where you're seeing spaces. In the Makefile on develop only `package:` is followed by space, so that's the odd one out. I think tab is better.

Reply:
I'll change back.